### PR TITLE
BZ2110179: Update UI procedures for top-level Virt

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.10+

Issue: [BZ2110179](https://bugzilla.redhat.com/show_bug.cgi?id=2110179)

Link to docs preview: [About OpenShift Virtualization](http://file.tlv.redhat.com/apinnick/BZ2110179-remove-workloads-ui/virt/about-virt.html) (Too many changes to point to a single section)

Additional information: I checked the UI against the [QE UI clusters](https://docs.google.com/spreadsheets/d/1BC_xl3lQity6M2IS71c1v6C1-AxErdMEOCTUT4544Uk/edit#gid=0) for 4.10 and 4.11.

### CODE REVIEW

File: modules/virt-editing-vm-yaml-web.adoc [hunk 1]
```diff
@@ -5,10 +5,12 @@
 //
 ifeval::["{context}" == "virt-edit-vms"]
 :object: virtual machine
+:object-gui: Virtual Machine
```
Comment (ousleyp) [inline modules/virt-editing-vm-yaml-web.adoc:8]: Should this be "VirtualMachines"?

File: modules/virt-importing-rhel-image-boot-source-web.adoc [hunk 2]
```diff
@@ -4,35 +4,30 @@
...
-* You must have access to the web server with the operating system image. For example: Red Hat Enterprise Linux web page with images.
 
-.Procedure
+* You must have access to a web page with the operating system image. For example: Download Red Hat Enterprise Linux web page with images.
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+.Procedure
 
-. Click the *Templates* tab.
+. In the {VirtProductName} console, click *Virtualization* -> *Templates* from the side menu.
```
Comment (ousleyp) [inline modules/virt-importing-rhel-image-boot-source-web.adoc:17]:
```suggestion
. In the {product-title} console, click *Virtualization* -> *Templates* from the side menu.
```

File: modules/virt-initiating-vm-migration-web.adoc [hunk 3]
```diff
@@ -10,20 +10,16 @@ Migrate a running virtual machine instance to a different node in the cluster.
 
 [NOTE]
 ====
-The *Migrate Virtual Machine* action is visible to all users but only admin users
-can initiate a virtual machine migration.
+The *Migrate* action is visible to all users but only admin users can initiate a virtual machine migration.
```
Comment (ousleyp) [inline modules/virt-initiating-vm-migration-web.adoc:13]:
```suggestion
The *Migrate* action is visible to all users, but only admin users can initiate a virtual machine migration.
```

File: modules/virt-initiating-vm-migration-web.adoc [hunk 4]
```diff
@@ -10,20 +10,16 @@ Migrate a running virtual machine instance to a different node in the cluster.
...
-selected virtual machine:
-** Click the Options menu {kebab} at the end of virtual machine and select
-*Migrate Virtual Machine*.
-** Click the virtual machine name to open the *Virtual Machine Overview*
-screen and click *Actions* -> *Migrate Virtual Machine*.
+. In the {VirtProductName} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+
+. You can initiate the migration from this page, which makes it easier to perform actions on multiple virtual machines on the same page, or from the *VirtualMachine details* page where you can view comprehensive details of the selected virtual machine:
+
+* Click the Options menu {kebab} at the end of virtual machine and select *Migrate*.
```
Comment (ousleyp) [inline modules/virt-initiating-vm-migration-web.adoc:22]:
```suggestion
* Click the Options menu {kebab} next to the virtual machine and select *Migrate*.
```

File: modules/virt-marking-vm-templates-favorites.adoc [hunk 5]
```diff
@@ -10,7 +10,6 @@ For easier access to virtual machine templates that are used frequently, you can
 
 .Procedure
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
-. Click the *Templates* tab.
-. Identify the Red Hat template that you want to mark as a favorite.
-. Click the Options menu {kebab} and select *Favorite template*. The template moves up higher in the list of displayed templates.
+. In the {VirtProductName} console, click *Virtualization* -> *Templates* from the side menu.
```
Comment (ousleyp) [inline modules/virt-marking-vm-templates-favorites.adoc:13]:
```suggestion
. In the {product-title} console, click *Virtualization* -> *Templates* from the side menu.
```

File: modules/virt-marking-vm-templates-favorites.adoc [hunk 6]
```diff
@@ -10,7 +10,6 @@ For easier access to virtual machine templates that are used frequently, you can
...
+
+. Identify the Red Hat template that you want to mark as a favorite and click the Star icon. The template moves up higher in the list of displayed templates.
```
Comment (ousleyp) [inline modules/virt-marking-vm-templates-favorites.adoc:15]: Not in the scope of this PR, but I'm confused; I don't see a star icon anywhere in the list of templates/VMs on the test system that I'm currently logged into.

File: modules/virt-template-fields-for-boot-source.adoc [hunk 7]
```diff
@@ -14,21 +15,21 @@ The following table describes the fields for *Add boot source to template* windo
...
+|Registry (creates PVC)
 |Specify the bootable operating system container that is located in a registry and accessible from the cluster. Example: kubevirt/cirros-registry-dis-demo.
 
 |Source provider
 |
 |Optional field. Add descriptive text about the source for the template or the name of the user who created the template. Example: Red Hat.
 
-.3+|Advanced
-|Storage class
+.3+|Advanced Storage settings
```
Comment (ousleyp) [inline modules/virt-template-fields-for-boot-source.adoc:31]: Is this referring to a UI element or should storage be lowercase?

File: modules/virt-viewing-vmi-ip-web.adoc [hunk 8]
```diff
@@ -6,12 +6,12 @@
...
 
-The IP information displays in the *Virtual Machine Overview* screen for the virtual machine.
+The IP information is displayed on the *VirtualMachine details* page for the virtual machine.
 
 .Procedure
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
-. Click the *Virtual Machines* tab.
-. Select a virtual machine name to open the *Virtual Machine Overview* screen.
+. In the {VirtProductName} console, click *Virtualization* -> *VirtualMachines* from the side menu.
```
Comment (ousleyp) [inline modules/virt-viewing-vmi-ip-web.adoc:13]:
```suggestion
. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
```

File: modules/virt-vm-rdp-console-web.adoc [hunk 9]
```diff
@@ -9,24 +9,18 @@
...
-* An RDP client installed on a machine on the same network as the
-Windows virtual machine.
+* An RDP client installed on a machine on the same network as the Windows virtual machine.
 
 .Procedure
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
-. Click the *Virtual Machines* tab.
-. Select a Windows virtual machine to open the *Virtual Machine Overview* screen.
+. In the {VirtProductName} console, click *Virtualization* -> *VirtualMachines* from the side menu.
```
Comment (ousleyp) [inline modules/virt-vm-rdp-console-web.adoc:22]:
```suggestion
. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
```

**Review by ousleyp — COMMENTED** [review]
Your changes LGTM, with a disclaimer that I didn't look at previews for every affected file. :) I left a handful of comments, most of which aren't referring to your changes themselves. 

This is incredibly helpful; thanks, Avital!

File: modules/virt-importing-rhel-image-boot-source-web.adoc (hunk 2 above)
Comment (apinnick) [reply modules/virt-importing-rhel-image-boot-source-web.adoc:17]: Totally agree. I hated seeing the Virt console everywhere. Will change this everywhere.

File: modules/virt-editing-vm-yaml-web.adoc (hunk 1 above)
Comment (apinnick) [reply modules/virt-editing-vm-yaml-web.adoc:8]: Yes! Good catch.

File: modules/virt-marking-vm-templates-favorites.adoc (hunk 6 above)
Comment (apinnick) [reply modules/virt-marking-vm-templates-favorites.adoc:15]: It's there, like when you mark a favorite in an app. The star is in the first column, to the left of the template name. It's empty by default. You click it to mark a favorite and the star turns yellow.

File: modules/virt-template-fields-for-boot-source.adoc (hunk 7 above)
Comment (apinnick) [reply modules/virt-template-fields-for-boot-source.adoc:31]: It's a UI element with poor capitalization. That's what the UI text says. If I had time, I would submit a bug.

File: modules/virt-marking-vm-templates-favorites.adoc (hunk 6 above)
Comment (apinnick) [reply modules/virt-marking-vm-templates-favorites.adoc:15]: I changed the text to this:

. Click the star icon next to a template to mark it as a favorite.
+
Favorite templates are displayed at the top of the template list.

**Pull request merged**